### PR TITLE
Implement hostOs detection for web target

### PR DESCRIPTION
**Pull request by eymar**
It's going to be used in compose to propertly configure KeyMapping and other os-specific features/configurations (potentially for scroll physics, etc).

### CODE REVIEW

File: skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt [hunk 1]
```diff
@@ -1,6 +1,10 @@
 package org.jetbrains.skiko
 
-actual val hostOs: OS = OS.JS
+import kotlinx.browser.window
+
+actual val hostOs: OS by lazy {
```
Comment (MatkovIvan) [inline skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt:5]: Why should we store (by lazy) the value?

Comment (MatkovIvan) [reply skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt]: i thought about `get() =`, but all variants will work here

File: skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package org.jetbrains.skiko
 
-actual val hostOs: OS = OS.JS
+import kotlinx.browser.window
+
+actual val hostOs: OS = detectHostOs()
```
[the remark this answers is not in the document]
Comment (eymar) [reply skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt:5]: Done. Also added a depracation for Arch.JS and Arch.WASM.
I decided to not detect the host Arch value for web for now, as I see no use cases for now.

File: skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt (hunk 1 above)
Comment (eymar) [reply skiko/src/jsMain/kotlin/org/jetbrains/skiko/OsArch.js.kt]: I decided to not add a getter to make sure the value is calculated only once. 
The value can potentially change when the app runs (by a client), but it's not a valid case to support. We (in compose) expect this value never changes.

**Review by igordmn — APPROVED** [review]

**Pull request merged**